ffmpeg_bitrate_stats/bitrate_stats.py

```diff
@@ -191,6 +199,10 @@ def _calculate_frame_sizes(self) -> list[FrameEntry]:
             "error",
             "-select_streams",
             self.stream_type[0] + ":0",
+            # "-analyzeduration",
+            # str(self.chunk_size*1000000),
```

Comment:
Please remove this if not needed.

Reply:
From my tests, it seems unnecessary; I copied it over from unleashlive's repo. I'll remove it.

As a side note, do you understand what the `-analyzeduration` option does? From what I read, it seems to specify the duration that ffprobe's probe attempts to read the input file's metadata/headers. Is that right?